packages/react-variant-view/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "react-variant-view",
+  "version": "0.1.0-beta.0",
+  "description": "cBioPortal Variant Viewer",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "jsnext:main": "dist/index.es.js",
+  "typings": "dist/index.d.ts",
+  "styles": "dist/styles.css",
+  "engines": {
+    "node": ">=8.12.0",
+    "yarn": ">=1.22.4"
+  },
+  "files": [
+    "dist"
+  ],
+  "author": "cBioPortal",
+  "license": "AGPL-3.0-or-later",
+  "repository": "cBioPortal/cbioportal-frontend",
+  "scripts": {
+    "build": "yarn run tcm && cross-env NODE_ENV=production NODE_OPTIONS=--max-old-space-size=2048 yarn run rollup",
+    "start": "yarn run watch",
+    "watch": "concurrently \"yarn run tcm:watch\" \"yarn run rollup:watch\"",
+    "watchSSL": "yarn run watch",
+    "rollup": "rollup -c rollup.config.ts",
+    "rollup:watch": "yarn run rollup --watch",
+    "tcm": "tcm -p src/**/*.module.scss",
+    "tcm:watch": "yarn run tcm --watch",
+    "prepare": "yarn run build",
+    "test": "cross-env CI=1 react-scripts-ts test --env=jsdom",
+    "test:watch": "yarn run test --watch"
+  },
+  "peerDependencies": {
+    "mobx": "^3.0.0 || ^4.0.0 || ^5.0.0",
+    "mobx-react": "^4.0.0 || ^5.0.0",
+    "prop-types": "^15.5.4",
+    "react": "^15.0.0 || ^16.0.0",
+    "react-dom": "^15.0.0 || ^16.0.0"
+  },
+  "dependencies": {
+    "autobind-decorator": "^2.1.0",
+    "bootstrap": "^4.3.1",
```

Not sure if it is a good idea to have direct dependency to bootstrap 4, since the main package still uses bootstrap 3. Maybe add this as <strong>peer dependency</strong> and let the consumer decide which version to use, something like:
```suggestion
    "bootstrap": "^3.0.0 || ^4.0.0",
```

---

good point